config/routes.rb

```diff
@@ -32,5 +27,18 @@
   resources :relationships, only: [:create, :destroy]
   get 'people', to: 'relationships#index'
 
+  resources :forgot_passwords, only: [:create]
+  get 'forgot_password', to: 'forgot_passwords#new'
+  get 'forgot_password_confirmation', to: 'forgot_passwords#confirm'
+
+  resources :password_resets, only: [:show, :create]
+  get 'expired_token', to: 'pages#expired_token'
+
+  resources :invitations, only: [:new, :create]
+
   get 'ui(/:action)', controller: 'ui'
+  
+  # Sidekiq monitoring
+  require 'sidekiq/web'
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.